src/model/kaniBinaryRunner.ts

```diff
@@ -129,16 +129,23 @@ async function catchOutput(command: string): Promise<number> {
 async function execLog(command: string): Promise<number> {
 	return new Promise((resolve, reject) => {
 		execAsync(command, (error, stdout, stderr) => {
-			if (error) {
+			if (stderr) {
+				// stderr is an output stream that happens when there are no problems executing the kani command but kani itself throws an error due to (most likely)
+				// a rustc error or an unhandled kani error
+				vscode.window.showErrorMessage('Kani Executable Crashed due to an underlying rustc error ->\n' + stderr);
```

Comment:
It looks like this error message goes into a window/dialog box rather than the terminal. I think this makes it a bit difficult to read. In the screenshot you posted, the `^^^^^` that indicate where an error is seem to be misaligned.

I wonder if there's a way to fix this? Does VSCode allow you to have leading whitespace and monospace font in these windows? Or alternatively, would you be able to print this message in a VSCode terminal instead of a window?

I'm not going to block this PR if this is difficult to do, but I do think it would be more helpful if stderr is properly formatted.

Reply:
I can look into that, and it's certainly something that needs to be improved but i haven't seen any examples yet of extensions with good error messaging. But I shall research more before concluding anything.

Also I have folded this PR into the other PR so I shall be closing this.